src/Plugin_Command.php

```diff
@@ -1195,9 +1225,15 @@ private function get_all_plugins() {
 	 *
 	 * @param array $args Passed-in arguments.
 	 * @param bool $all All flag.
+	 * @param string $exclude Comma separated list of plugin slugs.
 	 * @return array Same as $args if not all, otherwise all slugs.
 	 */
-	private function check_optional_args_and_all( $args, $all, $verb = 'install' ) {
+	private function check_optional_args_and_all( $args, $all, $verb = 'install', $exclude = null ) {
+		// Workaround for optional argument $verb
```

Comment:
This workaround shouldn't be needed, the argument has a default value of `'install'` already.